openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -30,15 +30,7 @@ <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
 
                     {% include "openassessmentblock/oa_submission_answer.html" with answer=student_submission.answer answer_text_label="Your response to the question above:" %}
 
-                    {% if allow_file_upload and file_url %}
-                    <h3 class="submission__answer__display__title">
-                        {% trans "Your Image" %}
-                    </h3>
-
-                    <div class="submission__answer__display__image">
-                        <img class="submission--image" alt="{% trans "The image associated with your submission." %}" src="{{ file_url }}"/>
-                    </div>
-                    {% endif %}
+                    {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url header="Your Upload" class_prefix="submission__answer" %}
```

Comment:
"show_warning" is an important setting.  Can we set it explicitly to false here if that is what is meant? And why is it OK to not show a warning here?